src/slider/Slider.js

```diff
@@ -547,21 +595,37 @@ Slider.defaultProps = {
 };
 
 const styles = StyleSheet.create({
-  container: {
+  containerHorizontal: {
     height: 40,
     justifyContent: 'center',
   },
+  containerVertical: {
+    width: 40,
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
   track: {
-    height: TRACK_SIZE,
     borderRadius: TRACK_SIZE / 2,
   },
+  trackHorizontal: {
+    height: TRACK_SIZE,
+  },
+  trackVertical: {
+    flex: 1,
+    width: TRACK_SIZE,
+  },
   thumb: {
     position: 'absolute',
     width: THUMB_SIZE,
     height: THUMB_SIZE,
     borderRadius: THUMB_SIZE / 2,
+  },
+  thumbHoizontal: {
```

Comment:
Small typo here. Great work with the PR. I've just stumbled upon this issue myself.